modules/Bio/EnsEMBL/Production/DBSQL/BulkFetcher.pm

```diff
@@ -390,6 +415,23 @@ sub get_transcripts {
     return $transcript_hash;
 } ## end sub get_transcripts
 
+
+sub set_cds {
+
+  my ($self , $exons_list, $transcript, $transcript_id ) = @_;
+
+  $log->debug("Getting CDS start and end cords for transcript"); 
```

Comment:
Not detailed enough I am afraid, can't you include in the log some actual data? 
Not to have this kind of log in the tests:
     
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data
2020/09/18 07:26:59 Getting CDS Data